Sources/XcodesKit/XcodeInstaller.swift

```diff
@@ -708,9 +708,10 @@ public final class XcodeInstaller {
     }
 
     func unarchiveAndMoveXIP(at source: URL, to destination: URL) -> Promise<URL> {
+        let xcodeExpansionDirectory = (try? Current.files.temporalDirectory(for: source)) ?? source.deletingLastPathComponent()
```

Comment:
#picky Can we move the `(try? Current.files.temporalDirectory(for: source)) ?? source.deletingLastPathComponent()` into it's own function inside of `Files` so that we can just call `try? Current.files.xcodeExpansionDirectory`

Reply:
I tried to solve this on https://github.com/RobotsAndPencils/xcodes/pull/179/commits/f1163aaf45111dcca1bc9a664fd8f49e7240ee3b. Please take a look! And don't worry about being picky 😊

Reply:
I mark this as resolved to remove some noise from this Pull Request, please fill free to reopen it if you have some comments!